erpc/networks_test.go

```diff
@@ -6068,6 +6068,37 @@ func TestNetwork_InFlightRequests(t *testing.T) {
 	})
 }
 
+func TestNetwork_SkippedUpstreams(t *testing.T) {
+	t.Run("ValidBlockNumberForFullNodeUpstream", func(t *testing.T) {
```

Comment:
Good to add other cases for archive nodes, and unknown node types (default behavior, which must be 'not' skip imo)